packages/web/components/funnels/concentrated-liquidity/learn-more.tsx

```diff
@@ -10,9 +10,24 @@ import {
   StepsIndicator,
 } from "~/components/stepper";
 
+const lotties: Record<string, Record<string, any>> = {};
```

Comment:
This is safe?

Reply:
It's pretty safe. There's no need for React to re-render, as we will consistently have the most recent result in the `loadLottie` function.